app/models/user.py

```
@@ -1,10 +1,30 @@
+"""사용자 모델.
+
+PostgreSQL users 테이블에 대응하는 SQLAlchemy 모델을 정의합니다.
+"""
+
 from sqlalchemy import Column, Integer, String, DateTime, func, Boolean
```

Flake8 (7.3.0)

[error] 'sqlalchemy.Column' imported but unused

(F401)